modules/Layout/Boilerplate/Boilerplate.module.scss

```diff
@@ -55,6 +55,7 @@
 
     display: flex;
     align-items: flex-start;
+    white-space: pre;
```

Comment:
I'm still not sure what happened here. The address string always had line breaks in it as far as I'm aware, so I wonder if this even worked before? And if it did, how would that be possible? 🤔